src/main/java/core/basesyntax/model/abstraction/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model.abstraction;
+
+public interface Draw {
+    String drawFigure();
```

Comment:
Could you read my comment once more:
![image](https://user-images.githubusercontent.com/20326858/200265527-c4c50d3b-8e84-4dca-85d9-5382993c379a.png)
Understanding abstraction is really important for development. You still have **Figure** in the interface (not in the class name, but in the method name).

One more issue:
![image](https://user-images.githubusercontent.com/20326858/200266412-5f051cd5-2e89-4e04-8e9c-577dd520d3b6.png)
Make this method void and use System.out.println as described in the task.

Reply:
I don't understand some things clearly with first time. I hope I will). Thanks for the mentorship!